transformer/events_creator.go

```diff
@@ -88,10 +84,9 @@ func (c *Transformer) createBeatEvents(fetchedResource fetching.Resource, metada
 	}
 
 	timestamp := time.Now()
-	resource := ResourceFields{
-		ID:   metadata.ResourceId,
-		Type: metadata.Type,
-		Raw:  fetcherResult.Resource,
+	resource := fetching.ResourceFields{
+		ResourceMetadata: metadata,
+		Raw:              fetcherResult.Resource,
```

Comment:
type and subtype?
or are they waiting for another PR?

Reply:
No, they are part of the `ResourceMetadata` type.
plz see `transformer.go`